content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2936,6 +2936,48 @@ model User {
 </tab>
 </TabbedContent>
 
+### <inlinecode>sequence()</inlinecode>
+
+<Admonition type="warning">
+
+**Only supported by CockroachDB** <br />
+The sequence function is for now only supported by [CockroachDB connector](/concepts/database-connectors/cockroachdb).
+
+</Admonition>
+
+Create a sequence of integers in the underlying database and assign the incremented values to the ID values of the created records based on the sequence.
+
+#### Optional arguments
+
+| Argument    | Example                                               | Description                                                                                                                                                                                                                                           |
+|-------------|-------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `virtual`   | `@default(sequence(virtual))`                         | Virtual sequences are sequences that do not generate monotonically increasing values and instead produce values like those generated by the built-in function `unique_rowid()`. They are intended for use in combination with `SERIAL`-typed columns. |
```

Comment:
What does "`SERIAL`-typed columns" mean here?

Reply:
We can drop that sentence, because it's one way to define autoincrementing columns on cockroachdb, but once the big PR is merged, we won't use it at all anymore.

Reply:
```suggestion
| `virtual`   | `@default(sequence(virtual))`                         | Virtual sequences are sequences that do not generate monotonically increasing values and instead produce values like those generated by the built-in function `unique_rowid()`. |
```